### ISSUE TITLE
The saved searches should have badges with the number of hits

### ISSUE THREAD

**Issue by Cito** [omnivore-app/omnivore#2749]
In the navigation, the menu items under "Saved Searches", like "inbox" should have badges with the number of items (at least optionally).

When using a read-it-later app I want to know if I pile up links too quickly, and how much I have to actually catch up.

You can look at Raindrop.io for inspiration which has this feature. See also #2479.

**Comment by jacksonh** [comment]
Yeah, because each one is an independent query we haven't done this yet as it'd ~5x the query costs depending on how many saved searches a person has. Once we have a caching layer for queries we could look at this.

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
